plone-4.3.x.cfg

```diff
@@ -1,2 +1,3 @@
 [buildout]
-extends = http://dist.plone.org/release/4.3.11/versions.cfg
+extends = http://dist.plone.org/release/4.3-latest/versions.cfg
```

Comment:
@tomgross please never use latest. If we do not pin specific versions the build will fail for reasons outside of the repo/code. This makes errors hard to track and fix.